crates/rune/src/runtime/bytes.rs

```diff
@@ -6,12 +6,13 @@ use crate::compile::{InstallWith, Named};
 use crate::runtime::{
     FromValue, Mut, RawMut, RawRef, RawStr, Ref, UnsafeFromValue, Value, VmError,
 };
+use serde::{Deserialize, Serialize};
 use std::cmp;
 use std::fmt;
 use std::ops;
 
 /// A vector of bytes.
-#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
```

Comment:
This should use a custom `Serialize` impl, since it can make use of [`Serialize::serialize_bytes`](https://docs.rs/serde/1.0.130/serde/trait.Serializer.html#tymethod.serialize_bytes) which for many formats can be dealt with more efficiently than sequences.

Reply:
@udoprog Fixed!